test/Feature/InsertSpec.hs

```diff
@@ -139,11 +149,11 @@ spec actualPgVersion = do
           }
 
       context "into a table with no pk" $ do
-        it "succeeds with 201 and a link including all fields" $ do
+        it "succeeds with 201 but no location header" $ do
           p <- post "/no_pk" [json| { "a":"foo", "b":"bar" } |]
```

Comment:
@wolfgangwalther I found a similar suggestion before, I just worked around it by renaming the variable. Like:
```suggestion
          resp <- post "/no_pk" [json| { "a":"foo", "b":"bar" } |]
```
And replace the `p` for `resp` in the lines that follow.

hlint is a bit lacking in this regard(see https://github.com/ndmitchell/hlint/issues/1009). So the workaround would be good.

Reply:
ok, i see - we're going to trick hlint here. I tried swapping two of those lines, let's see whether that works.

Reply:
I think it's also possible to specify a `.hlint.yaml` at the root of the project and ignore this suggestion. But not sure how to do it now.